src/app/compare/compare-buttons.tsx

```diff
@@ -31,6 +37,26 @@ export function findSimilarArmors(exampleItem: DimItem): CompareButton[] {
     !exampleItem.isExotic &&
     getIntrinsicArmorPerkSocket(exampleItem)?.plugged?.plugDef.displayProperties;
 
+  // exotic class item perks
+  const extraIntrinsicButtons =
+    (exampleItem.destinyVersion === 2 &&
+      filterMap(
+        getExtraIntrinsicPerkSockets(exampleItem),
+        (s) => s.plugged?.plugDef.displayProperties,
+      )?.map((intrinsic) => ({
+        buttonLabel: [
+          <BungieImage
+            key="1"
+            className={clsx(styles.intrinsicIcon, 'dontInvert')}
+            src={intrinsic.icon}
+          />,
+          intrinsic.name,
+          <ArmorSlotIcon key="slot" item={exampleItem} className={styles.svgIcon} />,
+        ],
+        query: `is:armor2.0 perk:${quoteFilterString(intrinsic.name)}`,
+      }))) ||
+    [];
```

Comment:
I'd like to reverse the order of this iterable so that the topmost perk is displayed first (since we reverse it again after building the list of suggestion buttons).
What's the best way to do that?

Reply:
You can just call `.reverse()` on it.

Reply:
Seems like it's considered an iterable not an array at this point so i'd have to wrap it first, I think. Is that worth doing?